cypress-studio/cypress/integration/checkout/shopping-cart.spec.js

```diff
@@ -0,0 +1,36 @@
+// shopping-cart.spec.js created with Cypress
+//
+// Start writing your Cypress tests below!
+// If you're unfamiliar with how Cypress works,
+// check out the link below and learn how to write your first test:
+// https://on.cypress.io/writing-first-test
+/* ==== Test Created with Cypress Studio ==== */
+it('can add a product into cart', function() {
+  /* ==== Generated with Cypress Studio ==== */
+  cy.visit('https://www.saucedemo.com/');
+  cy.get('[data-test="username"]').clear();
+  cy.get('[data-test="username"]').type('standard_user');
+  cy.get('[data-test="password"]').clear();
+  cy.get('[data-test="password"]').type('secret_sauce');
+  cy.get('[data-test="login-button"]').click();
+  cy.get('.title').should('be.visible');
+  cy.get('[data-test="add-to-cart-sauce-labs-backpack"]').click();
+  cy.get('.shopping_cart_link').click();
+  cy.get('.inventory_item_name').should('have.text', 'Sauce Labs Backpack');
+  cy.get('.cart_quantity').should('have.text', '1');
+  cy.get('.inventory_item_price').should('have.text', '$29.99');
```

Comment:
This will break if the price changes, is some websites the price might be dynamic. How about using a page object that doesn't change often?

Reply:
@Prasanth-V-K I will change them just to make sure if those elements were visible, sounds good?

Reply:
I don't tend to use Page Objects with Cypress.
I prefer in this case to save the price in an environment variable and assert against it.
So that if the price got changed, we can go and change the value of the environment variable.